src/qnngds/devices.py

```diff
@@ -3,8 +3,6 @@
 
 from phidl import Device
 
-# from phidl import quickplot as qp
-# from phidl import set_quickplot_options
```

Comment:
We kind of use qp and set_quickplot_options every time we create a new device to test it. I feel like commenting it is not such a bad idea otherwise you have to include, test your code and then delete it each time you create a new function. What do you think?

Reply:
I would argue that plotting of the device to test it should happen in a separate file and that you shouldn't run `python devices.py` to test your new device code (since it doesn't have a main method anyway). I would think you'd want to just `pip install -e` this repository (or even just build it with `python -m build` at the repository root) and test it with a separate short script somewhere else that imports `qnngds` and then creates your test device.

Reply:
ok fair enough